lib/ansible/module_utils/f5_utils.py

```diff
@@ -140,6 +140,10 @@ def fq_list_names(partition,list_names):
 
 # New style
 
+from abc import ABCMeta, abstractproperty
+from ansible.compat.six import with_metaclass
```

Comment:
@resmo  the compat version is used in several existing module utils (off which this one is based). Should those additionally be changed then?